docs/develop/php/versioning.mdx

```diff
@@ -0,0 +1,220 @@
+
+Instead, you'll need to deploy `postPatchActivity` and use the [Workflow::getVersion()](https://php.temporal.io/classes/Temporal-Workflow.html#method_getVersion) method to determine which version of the code to execute.
+
+```php
+#[WorkflowInterface]
+class MyWorkflow
+{
+    // ...
+
+    #[WorkflowMethod]
+    public function runAsync()
+    {
+        $version = yield Workflow::getVersion('Step 1', Workflow::DEFAULT_VERSION, 1);
+
+        $result = $version === Workflow::DEFAULT_VERSION
+            ? yield $this->activity->prePatchActivity()
+            : yield $this->activity->postPatchActivity();
+    }
+}
+```
+
+When `getVersion()` is run for the new Workflow execution, it records a marker in the Workflow history.
+All future calls to `GetVersion()` for this change Id (`Step 1` in the example) on this Workflow execution will always return the given version number.
+This is `1` in the example.
+
+:::note
+
+The Id passed to `getVersion` identifies the change.
+Each change is expected to have its own Id.
+If a change spawns multiple places in the Workflow code, and the new code should be either executed in all of them or in none of them, then they have to share the Id.
+
+:::
+
+If you make an additional change, such as replacing ActivityC with ActivityD, you need to add some additional code:
+
+```php
+#[WorkflowInterface]
+class MyWorkflow
+{
+    // ...
+
+    #[WorkflowMethod]
+    public function runAsync()
+    {
+        $version = yield Workflow::getVersion('Step 1', Workflow::DEFAULT_VERSION, maxSupported: 2);
+
+        $result = match($version) {
+            Workflow::DEFAULT_VERSION => yield $this->activity->prePatchActivity()
+            1 => yield $this->activity->postPatchActivity();
+            2 => yield $this->activity->anotherPatchActivity();
+        };
+    }
+}
+```
+
+Note that `maxSupported` changed from 1 to 2.
+A Workflow that had already passed this `GetVersion()` call before it was introduced will return `DEFAULT_VERSION`. 
+A Workflow that was run with `maxSupported` set to 1, will return 1.
+New Workflows will return 2.
+
+After you are sure that all of the Workflow executions prior to version 1 have completed, you can remove the code for that version. It should now look like the following:
+
+```php
+    #[WorkflowMethod]
+    public function runAsync()
+    {
+        $version = yield Workflow::getVersion('Step 1', minSupported: 1, maxSupported: 2);
+
+        $result = match($version) {
+            1 => yield $this->activity->postPatchActivity();
+            2 => yield $this->activity->anotherPatchActivity();
+        };
+    }
+```
+
+You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`. If an older version of the Workflow execution history is replayed on this code, it will fail because the minimum expected version is 1. After you are sure that all of the Workflow executions for version 1 have completed, then you can remove 1 so that your code would look like the following:
```

Comment:
```suggestion
You'll note that `minSupported` has changed from `DEFAULT_VERSION` to `1`.
If an older version of the Workflow Execution History is replayed on this code, it will fail because the minimum expected version is 1.
After you are sure that all of the Workflow executions for version 1 have completed, you can remove 1 so your code would look like the following:
```